packages/components/pie-modal/src/index.ts

```diff
@@ -208,6 +215,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
                 @click="${() => { this.isOpen = false; }}"
                 variant="ghost-secondary"
                 class="c-modal-closeBtn"
+                aria-label="${this.ariaLabels?.closeButton || 'Close'}"
```

Comment:
I don't think we should default to english copy here but keen to hear what others think

Reply:
I don't believe we should default to anything. users should pass the labels, and if they don't, we shouldn't render anything and don't make any assumptions.

Reply:
Fixed.